sandbox/qml.qrc

```diff
@@ -75,11 +75,8 @@
         <file>main.qml</file>
         <file>ThemeSwitch.qml</file>
         <file>examples/FilteringSorting.qml</file>
-        <file>images/RARI.png</file>
-        <file>images/SRToken.png</file>
         <file>demoapp/data/profile-image-1.jpeg</file>
         <file>demoapp/data/profile-image-2.jpeg</file>
-        <file>pages/StatusItemSelectorPage.qml</file>
```

Comment:
It was accidentally removed from sandbox pages during rebase (https://github.com/status-im/StatusQ/commit/e7b80a9ae2130fb50c1178849c496ed359ef601d). I'm going to restore it soon.

Reply:
I just removed it since it was a duplicate